webcomponents/tool/src/main/frontend/packages/sakai-conversations/src/SakaiConversationsSettings.js

```diff
@@ -50,8 +49,13 @@ export class SakaiConversationsSettings extends SakaiElement {
   }
 
   _saveGuidelines() {
+    // Using let because guidelines may be reassigned
+    let guidelines = this.querySelector("#settings-guidelines-editor")?.value || "";
 
-    const guidelines = this.querySelector("#settings-guidelines-editor")?.getContent();
+    // If guidelines are empty, use the sample guidelines
+    if (!guidelines.trim()) {
+      guidelines = this._i18n.community_guidelines_sample || "<p>Welcome to our community! Please follow these guidelines...</p>";
```

Comment:
This is a major code smell :)

shouldUpdate should just block the render 'til the i18n is ready